src/metabase/email.clj

```diff
@@ -41,10 +41,13 @@
 
 ;; ## PUBLIC INTERFACE
 
-(def ^{:arglists '([smtp-credentials email-details]), :style/indent 1} send-email!
+(defn send-email!
   "Internal function used to send messages. Should take 2 args - a map of SMTP credentials, and a map of email details.
    Provided so you can swap this out with an \"inbox\" for test purposes."
-  postal/send-message)
+  [smtp-credentials email-details]
+  (System/setProperty "mail.mime.splitlongparameters" "false") ; https://github.com/metabase/metabase/issues/11879#issuecomment-713816386
```

Comment:
there's an idiom to set this property once in task.clj:

```clojure
(when-not *compile-files*
  (System/setProperty "org.quartz.dataSource.db.connectionProvider.class" (.getName ConnectionProvider)))
```

I'm wondering if perhaps something like this is more what we want? Or is there a reason we continuously set this system property each time we send a message?

Reply:
Good point; added it to the toplevel and things are :ok_hand:

Reply:
awesome, this call of `System/setProperty` was my fault, I like this top-level approach.